pkg/crd/testdata/cronjob_types.go

```diff
@@ -360,6 +378,12 @@ type LongerString string
 // TotallyABool is a bool that serializes as a string.
 type TotallyABool bool
 
+// This tests string slice item validation.
+// +kubebuilder:validation:ItemMinLength=1
+// +kubebuilder:validation:ItemMaxLength=255
+// +kubebuilder:validation:ItemPattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?([.][a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
```

Comment:
Would it be possible to do something like this:
```suggestion
// +kubebuilder:validation:items:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?([.][a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
```

and then internally  re-use the existing validations rather than duplicating all of them?

Reply:
Ok, I've created #899

Reply:
@AlexanderYastrebov can you please update your existing PR rather than creating a new one for every change? Otherwise the whole discussion is spread up among a ton of PR and hard to follow-up on later on.

Reply:
I closed this and previous as they use different approaches.

Reply:
@AlexanderYastrebov that doesn't matter, please update the existing PR so there is one place that has the history of the discussion

Reply:
I can not reopen this closed PR fro some reason, I'll take a note for the next time I contribute.

Reply:
Ok, I think I managed to reopen it.